pinephone/lights/lights.cpp

```diff
@@ -68,6 +76,20 @@ static int sys_write_int(int fd, int value) {
     return amount == -1 ? -errno : 0;
 }
 
+static int sys_read_int(int fd, int& value) {
```

Comment:
Please rename to `sysfs_file_read_int` and use `const char *name` instead of `int fd` as an argument.